tests/e2e/manifests/chopconf/test-033-chopconf.yaml

```diff
@@ -0,0 +1,205 @@
+apiVersion: "clickhouse.altinity.com/v1"
+kind: "ClickHouseOperatorConfiguration"
+metadata:
+  name: "chop-config-01"
+spec:
```

Comment:
yuo don't need setup the full config here
just define sections which you will to change, operator will depp merge it with exists file config 
for example
```yaml
spec:
  clickhouse:
    access:
      schema: https
      port: 8443
```     
and remove comments, it unnecessary

Reply:
I have removed all unnecessary parts from chopconf.ymal